hw/ip/otp_ctrl/data/otp_ctrl_mmap.hjson

```diff
@@ -0,0 +1,194 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Use the translate-mmap.py script to update dependent files (like documentation
+// tables the comportable hjson and metadata SV package):
+//
+// $ cd ${PROJ_ROOT}/hw/ip/otp_ctrl/util/
+// $ ./translate-mmap.py
```

Comment:
do you think we should integrate this check into CI to make sure we've always generated the latest otp?

Reply:
yeah that is a good idea. let me add that.

Reply:
o that can be a different PR (or maybe just part of the makefile separately?)

Reply:
I'll add this in a followup up PR (the makefile is probably ok).